modules/representation_management/app/models/representation_management/form_2122_base.rb

```diff
@@ -122,6 +118,30 @@ def representative_phone
       end
     end
 
+    def veteran_state_code_truncated
+      veteran_state_code[0..1]
+    end
+
+    def claimant_state_code_truncated
+      claimant_state_code[0..1]
+    end
+
+    def veteran_zip_code_expanded
+      if veteran_zip_code_suffix.blank?
+        [veteran_zip_code[0..4], veteran_zip_code[5..8]]
+      else
+        [veteran_zip_code, veteran_zip_code_suffix]
```

Comment:
these should be truncated too now that the length validation is gone. Same for claimant. Same for 2122a rep zip.

Reply:
Done!